services/backend/src/persistence/environmental_authority.js

```diff
@@ -370,47 +370,5 @@ module.exports = (
           [geometriesConfig.tolerance, envAuthority],
         )
         .then((biomes) => biomes.rows[0].collection),
-
-    /**
-     * Get the coverage layer divided by categories in a given environmental authority
-     * @param {String} eaId environmental authority id
-     *
-     * @return {Object} Geojson object with the geometry
-     */
-    findCoverageLayer: (eaId) =>
```

Comment:
function `findAreaByCoverage` in this file should be deleted too

Reply:
Done!